### PR TITLE
Update Kingsoft logo

### PR DESCRIPTION
**Pull request by Anonymitaet**
Fix the issue of Kingsoft logo is not displayed

image

### CODE REVIEW

File: site2/docs/tiered-storage-aws.md [hunk 1]
```diff
@@ -47,14 +47,13 @@ This example uses Pulsar 2.5.1.
 
     **Output**
 
-    As shown from the output, Pulsar uses [Apache jclouds](https://jclouds.apache.org) to support [AWS S3](https://aws.amazon.com/s3/) and [GCS](https://cloud.google.com/storage/) for long term storage.
```
Comment (sijie) [inline site2/docs/tiered-storage-aws.md:50]: Why do you change this file?

Comment (Anonymitaet) [reply site2/docs/tiered-storage-aws.md:50]: It does not display correctly.

Comment (jiazhai) [reply site2/docs/tiered-storage-aws.md:50]: But this is not related with this change?

Comment (Anonymitaet) [reply site2/docs/tiered-storage-aws.md:50]: @jiazhai this change is not related to logo, just fix issues together.

**Pull request closed without merging**